components/GoldSponsors.js

```diff
@@ -11,7 +11,7 @@ export default function GoldSponsors({ className = '', showSupportBanner = true
             <img
             className="inline-block sm:h-16"
             src="/img/sponsors/redhat.svg"
-            alt="Red Hat"
+            alt="Link,Red Hat"
```

Comment:
Just like @akshatnema, I am wondering about the same. Why did you add the text "Link" to the `alt` text? When you read my [proposed changes](https://github.com/asyncapi/website/issues/735#issuecomment-1117620923), I mentioned that I wasn't sure about the third bullet point. I could as well be wrong about it completely. Therefore,  I suggest you revert to the original, unless you or someone else is sure the change is absolutely necessary.

Reply:
The same comment applies to the rest of the sponsors images.

Reply:
I reverted it, I thought since it contains a link as well as image so I added it. I wasn't sure too